tests/manage/pv_services/test_verify_rbd_based_rwo_pvc.py

```diff
@@ -274,10 +275,14 @@ def test_rbd_based_rwo_pvc(self, reclaim_policy):
 
         if reclaim_policy == "Delete":
             # Verify PV is deleted
-            pv_info = pv_obj.get(out_yaml_format=False)
-            if pv_info:
-                assert not (pv_name in pv_info), (
-                    f"PV {pv_name} exists after deleting PVC {pvc_obj.name}"
+            for pv_info in TimeoutSampler(
+                    30, 2, pv_obj.get, out_yaml_format=False
```

Comment:
this will throw CommandFailed if the PV is gone right?

Reply:
No, it is not getting a particular resource. 
Command will be :
oc -n namespace-test-0111501388 --kubeconfig /home/jijoy/jijoy-ocs-deployment/auth/kubeconfig get PersistentVolume

Reply:
Thanks